drivers/health_ip_followup_report/app/models/health_ip_followup_report/followups_report.rb

```diff
@@ -0,0 +1,50 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HealthIpFollowupReport
+  class FollowupsReport
+    def initialize(start_date:, end_date:)
+      @range = (start_date..end_date)
+    end
+
+    def qas_by_month
+      @qas_by_month ||= Health::QualifyingActivity.
+        submittable.
+        in_range(@range).
+        where(activity: :discharge_follow_up).
+        group(Arel.sql("DATE_TRUNC('MONTH', date_of_activity)")).
```

Comment:
You are limiting to a 3 month increment in the controller, but potentially, in the future, that could be expanded (or maybe someone can pick more than a year in the UI.  This only groups by month, so potentially you'll group 1/2020 with 1/2021 and 1/2022.  Might want to group by year and month.

Reply:
My mistake, date_trunc retains year, I was thinking of date_part.